frontend/src/pages/modelServing/screens/projects/ModelServingPlatformButtonAction.tsx

```diff
@@ -21,9 +21,9 @@ const ModelServingPlatformButtonAction: React.FC<ModelServingPlatformButtonActio
     <Button
```

Comment:
This button isn't triggering any action on click; there's a test that I believe expects this to open a modal (after having to update the test to get to the page where this button is rendered)

Reply:
Should be fixed now.